### PR TITLE
Molcas files for MP calculations

### PR DESCRIPTION
**Pull request by kunalsharma05**
Here are the input and output files for MOLCAS MP calculations using `&MBPT2` method.

### CODE REVIEW

File: data/Molcas/basicOpenMolcas18.0/water_mbpt2.in [hunk 1]
```diff
@@ -0,0 +1,17 @@
+&gateway
+ coord
+ 3
+
+ H    0.7493682    0.0000000    0.4424329
+ O    0.0000000    0.0000000   -0.1653507
+ H 	 -0.7493682   0.0000000    0.4424329
```
Comment (berquist) [inline data/Molcas/basicOpenMolcas18.0/water_mbpt2.in]: Please align this.

File: data/Molcas/basicOpenMolcas18.0/water_mbpt2.in [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+ basis = sto-3g
+ group = nosym
+
+&SEWARD
+>>>EXPORT MOCLAS_PRINT = 3
```
Comment (berquist) [inline data/Molcas/basicOpenMolcas18.0/water_mbpt2.in]: `MOCLAS -> MOLCAS`

**Comment by kunalsharma05** [comment]
@berquist I made the requested changes. Please review. Thanks!

**Comment by langner** [comment]
I have a general question: is there a substantial difference between OpenMolcas and the regular version that we have a `basicOpenMolcas` directory for the test files? That naming suggests that they are different programs.

**Comment by kunalsharma05** [comment]
@langner The output files are the same for both `Molcas` and `OpenMolcas`. This naming was suggested to me in the starting of the porject. `OpenMolcas` is a different program with different installation and usage, but the I/O files are the same for both programs.

**Comment by berquist** [comment]
Kunal is correct. The name changed to coincide with the open-sourcing, and apparently some features may be missing.

**Comment by langner** [comment]
> @langner The output files are the same for both Molcas and OpenMolcas. This naming was suggested to me in the starting of the porject. OpenMolcas is a different program with different installation and usage, but the I/O files are the same for both programs.

Functionality may be limited, but if the open version is a strict subset of the full version then in my mind they should share the unit test logfiles. In other words, if someone were to contribute files from the full version, I don't see the value of having separate `Molcas/basicMolcas*` directories.

**Pull request merged**